src/extension/background-script/actions/swaps/create.ts

```diff
@@ -0,0 +1,23 @@
+import Alby from "~/extension/background-script/connectors/alby";
+import state from "~/extension/background-script/state";
+import { MessageCreateSwap } from "~/types";
+
+const createSwap = async (message: MessageCreateSwap) => {
+  try {
+    const connector = (await state
```

Comment:
maybe we should add createSwap as an optional function to the connector interface, then we can do it without casting/fetching the Alby one specifically

Reply:
Good idea.